src/pages/paintings.js

```diff
@@ -1,8 +1,94 @@
 import * as React from "react"
+import { Composition, Box } from "atomic-layout"
+import styled from "styled-components"
 import { DefaultLayout } from "../components/DefaultLayout"
 
+import Section from "../components/Section"
+import { Button } from "../components/Button"
+import one from "../images/one.png"
+import two from "../images/two.png"
+import three from "../images/three.png"
+import four from "../images/four.png"
+
+const StyledBox = styled.div`
```

Comment:
I may be missing it, but `StyledBox` is not used in this file. Consider removing?

Reply:
It is used. This component separates text, title, and button from the images component and places his children in order.

Reply:
Yeap, you're right. On oversight on my part.